handlers/network.go

```diff
@@ -79,8 +79,10 @@ func Supply(w http.ResponseWriter, r *http.Request) {
 	}
 
 	depositContractBalanceGWei := decimal.NewFromBigInt(new(big.Int).SetBytes(addressMetadata.EthBalance.Balance), 0).DivRound(decimal.NewFromInt(params.GWei), 18)
+	// Deposit contract holds 320k LYX lost forever
+	depositContractFinalBalance := uint64(depositContractBalanceGWei.InexactFloat64()) - 320000000000000
```

Comment:
So this assumes that this 320k LYX is the only amount lost in this process - what about a single staker that would meet a similar fate? Do you think that this has been the only occurence of depositing keys for wrong network? Either way, this is still a good correction, but I doubt it's gonna be 100% accurate

Reply:
That's true. But it's the most important thing for us now, so we need to reduce by 320k first.